activemodel/lib/active_model/type/boolean.rb

```diff
@@ -1,9 +1,22 @@
 module ActiveModel
   module Type
-    class Boolean < Value # :nodoc:
+    # == Active \Model \Type \Boolean
+    #
+    # A class representing +boolean+ with functionality for coercion to and from
+    # a database or user input.
```

Comment:
This sentence feels way to clinical for lack of a better word, but I don't have a specific suggestion to improve it.